src/cmdlib.sh

```diff
@@ -263,38 +264,56 @@ prepare_git_artifacts() {
     local tarball="${1:?second argument me be the tarball name}"; shift;
     local json="${1:?third argument must be the json file name to emit}"; shift;
 
+    local is_dirty="unknown"
```

Comment:
can we remove this and below if/else statement since we aren't accepting an argument any longer?

Reply:
Yeah, dropped.